engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -253,10 +249,29 @@ public String search(@CommandParam("searched") String searched) {
      * @return List of blocks that match searched string
      */
     private List<String> findBlockMatches(String searchLowercase) {
-        return assetManager.getAvailableAssets(BlockFamilyDefinition.class)
-                .stream().<Optional<BlockFamilyDefinition>>map(urn -> assetManager.getAsset(urn, BlockFamilyDefinition.class))
-                .filter(def -> def.isPresent() && def.get().isLoadable() && matchesSearch(searchLowercase, def.get()))
-                .map(r -> new BlockUri(r.get().getUrn()).toString()).collect(Collectors.toList());
+        ResourceUrn curUrn;
+        List<String> l=new ArrayList<String>();
```

Comment:
We try and have all variables have useful names, even if they're not used much. Here I'd suggest something like `matchingBlocks` or even just `outputList`

Reply:
Fixed! Will keep this in mind from now on .